specifyweb/frontend/js_src/lib/components/WbPlanView/LineComponents.tsx

```diff
@@ -147,7 +151,18 @@ export function MappingLineComponent({
       aria-labelledby={id('header')}
       className="contents"
     >
-      <div className="border-t border-t-gray-500 py-2 print:hidden">
+      <div className="flex gap-2 border-t border-t-gray-500 py-2 print:hidden">
+        {typeof handleRemoveLine === 'function' && (
```

Comment:
according to your typescript type, onRemoveLine is always defined, so why this check in here?